pylint_django/transforms/foreignkey.py

```diff
@@ -106,6 +108,9 @@ def infer_key_classes(node, context=None):
                     # 'auth.models', 'User' which works nicely with the `endswith()`
                     # comparison below
                     module_name += '.models'
+                except ImproperlyConfigured:
+                    raise RuntimeError("DJANGO_SETTINGS_MODULE required for resolving ForeignKey "
+                                       "string references, see Usage section in README!")
```

Comment:
I think adding a URL to the README would be good as a slight QoL improvement for anyone reading the error message, though I'd prever https://pypi.org/project/pylint-django/ rather than GitHub just in case pylint-django ever moves off GitHub (can happen, pylint used to be on bitbucket!)

Reply:
Added as part of #283